src/Native/Windows/CMakeLists.txt

```diff
@@ -97,8 +94,6 @@ endif()
 
 if (__LinkCustomCRT STREQUAL "true" AND $ENV{__BuildArch} STREQUAL "arm" )
     set(__LinkArgs "${__LinkArgs} /DEFAULTLIB:ole32.lib")
-elseif(__LinkCustomCRT STREQUAL "true" AND NOT $ENV{__BuildArch} STREQUAL "arm" )
-    set(__LinkArgs "${__LinkArgs} /NODEFAULTLIB")
 endif()
 
```

Comment:
The `_tools_BinscopeFilter_packages_Newtonsoft.Json.9.0.1_lib_net45_Newtonsoft.Json.txt` binary file is still here.

Reply:
It shows on my github as being removed, which I'd like to do even though it isn't technically related to this PR

Reply:
ok